client/trino-cli/src/main/java/io/trino/cli/Console.java

```diff
@@ -147,7 +148,7 @@ public boolean run()
         Runtime.getRuntime().addShutdownHook(new Thread(() -> {
             exiting.set(true);
             interruptor.interrupt();
-            awaitUninterruptibly(exited, EXIT_DELAY.toMillis(), MILLISECONDS);
+            verify(awaitUninterruptibly(exited, EXIT_DELAY.toMillis(), MILLISECONDS));
```

Comment:
Is this a bug in error prone? I'd rather disable that check until it gets fixed instead of polluting the code with code just to make the checker happy.

Reply:
It's not. It's a guava change: https://github.com/google/guava/commit/0ef6688750d6432e42830dbe5b522cf8c8182790 (related: https://github.com/google/guava/issues/3655)

Reply:
This is an explicit choice to require checking the return value: https://github.com/google/guava/commit/aedb0efb151c0c27a2d4f88a91018e25d589039d

Note that I don't think `verify()` is correct here. We actually want to continue to ignore the return value and close the terminal, even if the main thread hasn't exited. So the correct approach is to suppress the check.